File: app/futurehome-kotlin/io/sentry/android/core/MainLooperHandler.java
package io.sentry.android.core;

import android.os.Handler;
import android.os.Looper;

final class MainLooperHandler {
   private final Handler handler;

   MainLooperHandler() {
      this(Looper.getMainLooper());
   }

   MainLooperHandler(Looper var1) {
      this.handler = new Handler(var1);
   }

   public Thread getThread() {
      return this.handler.getLooper().getThread();
   }

   public void post(Runnable var1) {
      this.handler.post(var1);
   }
}

File: app/futurehome-kotlin/io/sentry/android/replay/util/MainLooperHandler.kt
package io.sentry.android.replay.util

import android.os.Handler
import android.os.Looper

internal class MainLooperHandler(looper: Looper = Looper.getMainLooper()) {
   public final val handler: Handler

   fun MainLooperHandler() {
      this(null, 1, null);
   }

   init {
      this.handler = new Handler(var1);
   }

   public fun post(runnable: Runnable) {
      this.handler.post(var1);
   }
}
